### PR TITLE
Improved search capabilties on manage users page.

### PR DESCRIPTION
**Pull request by adobrusky**
Added searching by first name, last name, expiration date, and email. Fixed searching by tool authorization. Also added the database connection information to the config.json.

### CODE REVIEW

**Comment by adobrusky** [comment]
Here is a link to the documentation for review/testing purposes: https://github.com/cseseniordesign/reservations/wiki/Story-%237-Search-Member-Table-%7C-User-and-Technical-Documentation

File: routes/admin/users.rb [hunk 1]
```diff
@@ -46,31 +55,71 @@
 get '/admin/users/?' do
     @breadcrumbs << {:text => 'Admin Users'}
 
+    first_name = params[:first_name]
+    last_name = params[:last_name]
+    email = params[:email]
     studio_status = params[:studio_status]
-    selected_tool = params[:selected_tool]
```
Comment (evanmielak) [inline routes/admin/users.rb:50]: Why did you remove this? What was the purpose before?

**Review by evanmielak — APPROVED** [review]
